packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -267,65 +263,49 @@ export class FileStorage implements DStore {
       await fs.unlink(fpath);
     }
 
-    // if have children, keep this node around as a stub
+    // if have children, create stub note with a new id
     if (!_.isEmpty(noteToDelete.children)) {
```

Comment:
do the children need to add the new stub note as their parent id?

Reply:
Good catch. Fixed and updated test cases